fastmri_recon/models/subclassed_models/ncpdnet.py

```diff
@@ -38,8 +40,8 @@ def __init__(
             multi_gpu=True,
             **kwargs,
         )
-        self.op = NFFT(im_size=self.im_size, multicoil=self.multicoil)
-        self.adj_op = AdjNFFT(im_size=self.im_size, multicoil=self.multicoil, density_compensation=dcomp)
+        self.op = NFFT(im_size=self.im_size, multicoil=self.multicoil, grad_traj=grad_traj)
+        self.adj_op = AdjNFFT(im_size=self.im_size, multicoil=self.multicoil, density_compensation=dcomp, grad_traj=grad_traj)
```

Comment:
```suggestion
        self.op = NFFT(im_size=self.im_size, multicoil=self.multicoil, grad_traj=self.grad_traj)
        self.adj_op = AdjNFFT(im_size=self.im_size, multicoil=self.multicoil, density_compensation=dcomp, grad_traj=self.grad_traj)
```

I know in this case it doesn't make much of a difference, but I prefer it this way, in case one day someone changes the `self.grad_traj` in between.